posthog/auth.py

```diff
@@ -96,7 +96,7 @@ def authenticate(self, request: Request):
                 )
         if request.GET.get("temporary_token"):
             User = apps.get_model(app_label="posthog", model_name="User")
-            user = User.objects.filter(temporary_token=request.GET.get("temporary_token"))
+            user = User.objects.filter(is_active=True, temporary_token=request.GET.get("temporary_token"))
```

Comment:
Given this endpoint seems like it would get called a lot - does this have an appropriate index?

Reply:
Perfect, thanks verifying this!